plugins/chat/app/services/chat/lookup_channel_threads.rb

```diff
@@ -97,7 +97,10 @@ def fetch_threads(guardian:, channel:, **)
             user: :user_status,
           ],
         )
-        .joins(:user_chat_thread_memberships, :last_message, :original_message)
+        .joins(:user_chat_thread_memberships, :original_message)
+        .joins(
```

Comment:
I have to do this because we need `last_message` on the sort, and if you just do joins with the symbol it gets some other alias from AR.